documentation/createmodal-order.md

```
@@ -0,0 +1,237 @@
+# CreateModal ordering
+
+### Table of contents
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)